### DISCUSSION TITLE
`sphinx.ext.doctest` / doctest support

### DISCUSSION THREAD

**Discussion by tony** [executablebooks/MyST-Parser#601] [Ideas]
[`sphinx.ext.doctest`](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html) (source: [sphinx/ext/doctest.py](https://github.com/sphinx-doc/sphinx/blob/5.x/sphinx/ext/doctest.py))

It looks like, at the moment, doctests aren't parsed. This would be really nice to add (and potentially to dogfood for both coverage and documentation purposes)

Related: https://github.com/Erotemic/xdoctest/issues/68, https://github.com/thisch/pytest-sphinx/issues/30

Also referenced [here](https://github.com/executablebooks/MyST-Parser/issues/444#issuecomment-987296252) as of today at v0.18.0 these are parsed as [`nodes.literal_block`](https://pydoc.dev/docutils/latest/docutils.nodes.literal_block.html) not [`nodes.doctest_block`](https://pydoc.dev/docutils/latest/docutils.nodes.doctest_block.html).

**Comment by cjw296** [comment]
I'm in the process of adding MyST support to [Sybil](https://sybil.readthedocs.io/), which should cover this use case.

**Reply by chrisjsewell** [reply]
sounds good, did you get anywhere?

**Reply by cjw296** [reply]
Yep, MyST fully supported now: https://sybil.readthedocs.io/en/latest/myst.html

**Reply by chrisjsewell** [reply]
🥳
